internal/server/handlers/handlers.go

```diff
@@ -69,27 +104,42 @@ func ListAllMetrics(storage storage.Storager) http.HandlerFunc {
 
 func PlainGetMetricHandler(storage storage.Storager) http.HandlerFunc {
 	return func(res http.ResponseWriter, req *http.Request) {
+		ctx := req.Context()
+
+		log, err := logger.FromContext(ctx)
+		if err != nil {
+			http.Error(res, "internal logger error", http.StatusInternalServerError)
+			return
+		}
+
 		metricType := chi.URLParam(req, "type")
 		metricName := chi.URLParam(req, "name")
 		if metricType == "" {
+			log.Errorf("Missing metric type")
```

Comment:
```suggestion
			log.Error("Missing metric type")
```
Если в строку, которую должен распечатать логгер, не подставляется хотя бы одна переменная, имеет смысл отказаться от использования в этом случае функций с суффиксом `f`. Они весьма тяжеловесны. Лучше описать по два комплекта на каждый log-level: Info + Infof, Debug + Debugf - и использовать то, что соответствует вызову.